spec/models/intake_spec.rb

```diff
@@ -375,6 +375,19 @@
         expect(intake.student_names).to eq([])
       end
     end
+
+    context "when there is no spouse verified but the spouse was a student" do
+      let(:intake) do
+        create :intake, was_full_time_student: "yes", spouse_was_full_time_student: "yes"
+      end
+      before do
+        create :user, intake: intake, first_name: "Henrietta", last_name: "Huckleberry"
+      end
+
+      it "Shows a placeholder for the spouse name" do
+        expect(intake.student_names).to eq(["Henrietta Huckleberry", "Your spouse"])
+      end
```

Comment:
nit: spacing before `before` and the capitalized "Shows"

Reply:
done, thanks!